Gemfile

```diff
@@ -1,3 +1,10 @@
 source 'https://rubygems.org'
 
 gemspec
+
+group :development do
```

Comment:
I don't know @tomas-stefano, that feels even less appropriate!

If I'm not mistaken, some of these gems have no relation to the actual application and are only wanted on the occasion that you're doing profiling, so feels like they should be kept out at all times other than when you're doing that.

I was wondering if there was some sort of solution along the lines of giving them a new group such as 'profiling' then running the profile task with `RAILS_GROUPS=profiling ruby -Ilib bin/markdown-libraries-benchmark`?

I've not done this before myself so might not work, just have seen it done elsewhere and feels like it would!